test/driver/error_embedding/test.t/run.t

```diff
@@ -7,13 +7,13 @@ With the `-embed-errors` options, if a PPX raises, the first such exception
 is caught and packed into an output AST
 
   $ echo "let _ = [%raise]" > impl.ml
-  $ raiser -embed-errors impl.ml
+  $ ./../raiser.exe -embed-errors impl.ml
```

Comment:
Isn't `../raiser.exe` enough here? Same for the other occurences in this file and some of the others!

Reply:
Good catch, thanks!